draft-ietf-rtcweb-jsep.xml

```diff
@@ -4128,47 +4128,45 @@ AliceUA->BobUA:     audio+video+data sent from Alice to Bob
         <t>
           <figure>
             <artwork alt="offer-B1">
-              <![CDATA[
+<![CDATA[
 v=0
 o=- 4962303333179871723 1 IN IP4 0.0.0.0
 s=-
 t=0 0
-a=group:BUNDLE a1 d1
 a=ice-options:trickle
+a=group:BUNDLE a1 d1
+
 m=audio 9 UDP/TLS/RTP/SAVPF 96 0 8 97 98
 c=IN IP4 0.0.0.0
-a=rtcp:9 IN IP4 0.0.0.0
 a=mid:a1
-a=msid:57017fee-b6c1-4162-929c-a25110252400
-       e83006c5-a0ff-4e0a-9ed9-d3e6747be7d9
 a=sendrecv
 a=rtpmap:96 opus/48000/2
 a=rtpmap:0 PCMU/8000
 a=rtpmap:8 PCMA/8000
 a=rtpmap:97 telephone-event/8000
 a=rtpmap:98 telephone-event/48000
 a=maxptime:120
-a=ice-ufrag:ATEn1v9DoTMB9J4r
+a=extmap:1 urn:ietf:params:rtp-hdrext:sdes:mid
+a=extmap:2 urn:ietf:params:rtp-hdrext:ssrc-audio-level
+a=msid:57017fee-b6c1-4162-929c-a25110252400
+       e83006c5-a0ff-4e0a-9ed9-d3e6747be7d9
+a=ice-ufrag:ATEn
 a=ice-pwd:AtSK0WpNtpUjkY4+86js7ZQl
 a=fingerprint:sha-256
-              19:E2:1C:3B:4B:9F:81:E6:B8:5C:F4:A5:A8:D8:73:04
-             :BB:05:2F:70:9F:04:A9:0E:05:E9:26:33:E8:70:88:A2
+              29:E2:1C:3B:4B:9F:81:E6:B8:5C:F4:A5:A8:D8:73:04:
```

Comment:
Example 7.1 uses the 19: cert; I wanted all the values in the samples to be distinct to avoid any confusion, so I changed the first octet to 29 in this sample.